sdk/identity/identity/src/msal/browserFlows/msalAuthCode.ts

```diff
@@ -190,16 +190,19 @@ To work with multiple accounts for the same Client ID and Tenant ID, please prov
           "Silent authentication failed. We couldn't retrieve an active account from the cache.",
       });
     }
+   
 
     const parameters: msalBrowser.SilentRequest = {
       authority: options?.authority || this.msalConfig.auth.authority!,
       correlationId: options?.correlationId,
       claims: options?.claims,
-      account: publicToMsal(account),
       forceRefresh: false,
       scopes,
     };
-
+    if(publicToMsal(account)!==null){
```

Comment:
`publicToMsal` can't return null can it? Either it throws or returns an object

Reply:
This needs to be cleaned up